src/pal/tools/gcc-compiler-override.txt

```diff
@@ -6,7 +6,7 @@ SET (CLR_C_FLAGS_CHECKED_INIT          "-g -O2")
 SET (CMAKE_C_FLAGS_RELEASE_INIT        "-g -O3")
 SET (CMAKE_C_FLAGS_RELWITHDEBINFO_INIT "-g -O2")
 
-SET (CMAKE_CXX_FLAGS_INIT                "-Wall  -Werror=conversion-null -std=c++11")
+SET (CMAKE_CXX_FLAGS_INIT                "-Wall  -Werror=conversion-null -std=c++11 -msse3")
```

Comment:
We do not have SSE3 as min-bar on Clang. Why is it a good idea to have SSE3 as minbar for GCC?

Reply:
Let me test without. I was just copying from a web link. The website mentions that in order to use __m128 types, we need to enable -msse3 compiler option.

Reply:
I just realized that PAL is defining its own __m128 data type and intrinsics. I'll revert.